Kernel/KString.cpp

```diff
@@ -6,6 +6,8 @@
 
 #include <Kernel/KString.h>
 
+extern bool early_boot;
```

Comment:
Maybe `g_in_early_boot` to be consistent with the rest of the codebase?

Reply:
None of the globals in `init.cpp` follow that convention, but sure! Happy to accommodate my review overlords.

Reply:
done

Reply:
I'll annoy people with it until they're all converted to the project code style then :P